### PR TITLE
License extension 

### LINKED ISSUE #1141: License menu item should come from an extension
Reason: visible license/eula only applies to built binary distribution.

### PR DESCRIPTION
**Pull request by ixrock**
close #1141

### CODE REVIEW

File: extensions/license/main.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { LensMainExtension } from "@k8slens/extensions";
+import { shell } from "electron";
```
Comment (ixrock) [inline extensions/license/main.ts]: This needs to double check. 
It's compiled to `require("electron")` in `main.js` but how it's eventually consumed inside Lens runtime with such call I'm not sure (works well in `development`-mode though).
Currently after build with `yarn build:mac` and running `Lens.app` extensions declared in `package.json ` not loaded at all..

Comment (ixrock) [reply extensions/license/main.ts]: Seems like this is broken in `master` as well.

**Steps to reproduce**:
1. `make build-extensions`
1. `yarn build:mac`
1. Run `dist/mac/Lens.app`

image

Comment (jim-docker) [reply extensions/license/main.ts]: > This needs to double check.
> It's compiled to `require("electron")` in `main.js` but how it's eventually consumed inside Lens runtime with such call I'm not sure (works well in `development`-mode though).
> Currently after build with `yarn build:mac` and running `Lens.app` extensions declared in `package.json ` not loaded at all..

Could wrap and expose `shell. openExternal()` through the extensions api?

Comment (jim-docker) [reply extensions/license/main.ts]: > Seems like this is broken in `master` as well.
> 
> **Steps to reproduce**:
> 
> 1. `make build-extensions`
> 2. `yarn build:mac`
> 3. Run `dist/mac/Lens.app`
> 
seems to be fixed...

**Review by jim-docker — APPROVED** [review]

File: extensions/license/main.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+export default class LicenseLensMainExtension extends LensMainExtension {
+  appMenus = [
+    {
+      parentId: "help",
+      label: "License",
+      async click() {
+        shell.openExternal("https://k8slens.dev/licenses/eula.md");
```
Comment (jakolehm) [inline extensions/license/main.ts]: Would be safer if we expose `openExternal` (or whole `shell`) via extension-api (maybe via `Util` namespace?).

**Review by jakolehm — CHANGES_REQUESTED** [review]

File: extensions/license/main.ts (hunk 1 above)
Comment (ixrock) [reply extensions/license/main.ts]: Don't know what I'm doing wrong or is it some cached files.. but this is still happening on my machine (even after `master` merge)

**Comment by ixrock** [comment]
@jakolehm PTAL, added helper `Util.openExternal` to extensions-api

**Review by jakolehm — CHANGES_REQUESTED** [review]
We need to keep LICENSE file in the root folder.

**Comment by ixrock** [comment]
@jakolehm could be merged already?

**Review by jakolehm — APPROVED** [review]

**Pull request merged**